fabric/src/main/java/net/minescript/common/Minescript.java

```diff
@@ -3981,6 +4010,116 @@ private static Optional<String> handleScriptFunction(
+          HandledScreen handledScreen = (HandledScreen) screen;
+          ScreenHandler screenHandler = handledScreen.getScreenHandler();
+          if (screenHandler.slots.size() <= slot) {
+            return Optional.empty();
+          }
+          ItemStack itemStack = screenHandler.slots.get(slot).getStack();
+          return Optional.of(itemStackToJsonString(itemStack, OptionalInt.of(slot), false));
+        } else {
+          return Optional.empty();
+        }
+      }
+
+      case "container_get_items": // List of Items in Chest
+      {
+        Screen screen = minecraft.currentScreen;
+        if (screen instanceof HandledScreen<?>) {
+          HandledScreen handledScreen = (HandledScreen) screen;
+          ScreenHandler screenHandler = handledScreen.getScreenHandler();
+          Slot[] items = screenHandler.slots.toArray(new Slot[0]);
+          StringBuilder resultString = new StringBuilder("[");
+          for (int i = 0; i < items.length; i++) {
+            ItemStack itemStack = items[i].getStack();
+            if (itemStack.isEmpty()) {
+              continue;
+            }
+            resultString.append(itemStackToJsonString(itemStack, OptionalInt.of(items[i].id), false));
+            if (i < items.length - 1) {
+              resultString.append(",");
+            }
+          }
+          resultString.append("]");
+          return Optional.of(resultString.toString());
+        } else {
+          return Optional.empty();
+        }
+      }
+
+      case "container_click_slot": // Click on slot in container
+      {
+        if (args.size() != 1) {
+          logUserError("Error: `{}` expected 1 param (slot id) but got: {}", functionName, argsString);
+          return Optional.empty();
+        }
+        Screen screen = minecraft.currentScreen;
+        int slot = getStrictIntValue(args.get(0)).getAsInt();
+        if (screen == null) {
+          return Optional.of("false");
+        }
+        if (screen instanceof HandledScreen<?>) {
+          HandledScreen handledScreen = (HandledScreen) screen;
+          ScreenHandler screenHandler = handledScreen.getScreenHandler();
+          if (screenHandler.slots.size() <= slot) {
+            logUserError("Error: `{}`: slot {} does not exist", functionName, slot);
+            return Optional.of("false");
+          }
+          //screenHandler.onSlotClick(slot, 0, SlotActionType.PICKUP, minecraft.player);
+          //screenHandler.sendContentUpdates();
+          PacketByteBuf buf = new PacketByteBuf(Unpooled.buffer());
+          buf.writeByte(screenHandler.syncId);
+          buf.writeShort(slot);
+          buf.writeByte(0); // Button (0 for left click)
+          buf.writeShort(0); // Action type (0 for click)
+          buf.writeShort(0); // Mode (0 for normal)
+          buf.writeItemStack(screenHandler.getSlot(slot).getStack()); // Item stack
+          minecraft.getNetworkHandler().sendPacket(new ClickSlotC2SPacket(buf));
```

Comment:
Please add a variable `connection` like this:

```
var connection = minecraft.getNetworkHandler();
connection.sendPacket(new ClickSlotC2SPacket(buf));
```

That patten will allow `tools/rename_minecraft_symbols.py` to translate these symbols to the symbols used for the Forge mod.